### ISSUE TITLE
5$: Error on stray comma after methods in classes

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#374]
```js
class IDLExtractor {
    constructor() {
        this._inIDL = false;
    }

    onopentag(tag, attributes) {
        if (tag === "code") {
            if (attributes.class === "idl") {
                this._inIDL = true;
            }
        }
    }

    ontext(text) {
        if (this._inIDL) {
            process.stdout.write(text);
        }
    }, // ERROR
}
```

**Comment by dav000** [comment]
I am interested in working on this and have a potential fix.

```
$ ./build/quick-lint-js ./f.js
./f.js:18:6: error: unexpected token [E054]
```

**Comment by strager** [comment]
> I am interested in working on this and have a potential fix.
> 
> ```
> $ ./build/quick-lint-js ./f.js
> ./f.js:18:6: error: unexpected token [E054]
> ```

I'd prefer a more helpful message, such as "commas are not allowed between class methods".

**Comment by dav000** [comment]
Sounds good. 

> $ ./build/quick-lint-js f.js
> f.js:18:6: error: commas are not allowed between class methods [E210]
